css/properties/scroll-margin.json

```
@@ -15,12 +15,26 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": {
-              "version_added": "68"
-            },
-            "firefox_android": {
-              "version_added": "68"
-            },
+            "firefox": [
+              {
+                "version_added": "90"
+              },
+              {
+                "version_added": "68",
```

If we're going with two support statements, then this one needs to be closed out with (sorry, should've mentioned this before):

```suggestion
                "version_added": "68",
                "version_removed": "90",
```